beacon-chain/state/stategen/cold_test.go

```diff
@@ -97,11 +98,11 @@ func TestLoadColdStateByRoot_CanGet(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	loadedState, err := service.loadColdStateByRoot(ctx, blkRoot)
+	loadedState, err := service.StateByRoot(ctx, blkRoot)
```

Comment:
Please rename this test. It is currently `TestLoadColdStateByRoot_CanGet`, but now only testing StateByRoot.